orc8r/cloud/helm/orc8r/charts/metrics/values.yaml

```diff
@@ -285,10 +285,14 @@ userGrafana:
 
   volumes:
     # Default volume configurations for grafana data.
-    dashboards: {}
-    datasources: {}
-    dashboardproviders: {}
-    grafanaData: {}
```

Comment:
There is a specific reason why these keys are like this: https://github.com/magma/magma/commit/fd3178ca14e3c3f401f4c50942a3b838fc3b3746